lexers/t/tea.go

```diff
@@ -0,0 +1,19 @@
+package t
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// Tea lexer. Lexer for Tea Templates <http://teatrove.org/>.
+var Tea = internal.Register(MustNewLexer(
```

Comment:
It's named `TeaTemplate` in pygments. https://github.com/pygments/pygments/blob/a590ac5ea7c00a41e253834306bfa19e38349c0b/pygments%2Flexers%2Ftemplates.py#L1646

Reply:
Forget about it.